src/global.h

```diff
@@ -319,11 +319,41 @@ typedef unsigned char      uint8_t;
 // definition for custom event
 #define MS_PACKET_RECEIVED 0
 
-/* Classes ********************************************************************/
+/* Exception Classes **********************************************************/
+
+class CInfoExit
+{
+public:
+    CInfoExit ( QString strMsg ) : strInfoMsg ( strMsg ) {}
+
+    QString GetInfoMessage() const { return strInfoMsg; }
+
+protected:
+    QString strInfoMsg;
+};
+
+class CErrorExit
```

Comment:
> I'd call this something like `CCmdLineErr`, to indicate what it's actually being used to report.

Again, this is a 'universal' solution, so not only usable for commandline errors, so why name it that way just because we now use it for commandline errors?

Reply:
I assume we can leave it as is - if we use it in future (and I think there is work coming up by @pgScorpio which uses it)